src/scripts/syncBreaches.js

```diff
@@ -11,7 +11,83 @@
 
 import { req, formatDataClassesArray } from '../utils/hibp.js'
 import { getAllBreaches, upsertBreaches } from '../db/tables/breaches.js'
+import { mkdir, readdir } from "node:fs/promises";
+import { resolve as pathResolve } from "node:path";
+import { finished } from 'stream/promises';
+import fs from "fs";
```

Comment:
Self note since this was semi-confusing to me initially (re: why we need "node:fs/promises" (L14) and "fs"):

- `fs.existsSync()` L59
- `fs.createWriteStream()` L77
- `fs.promises.mkdir()` L59
- `fs.promises.readdir()` L62

Not sure if we could possibly use [`fs.mkdirSync()`](https://nodejs.org/dist/latest-v18.x/docs/api/fs.html#fsmkdirsyncpath-options) and [`fs.readdirSync()`](https://nodejs.org/dist/latest-v18.x/docs/api/fs.html#fsreaddirsyncpath-options) and then not need the `fs:promises` async versions.

nit: can we prefix this with "node:fs" to be consistent with L14-15 above?